### PR TITLE
Version changes

### PR DESCRIPTION
**Pull request by mrajashree**

### CODE REVIEW

File: rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -1387,6 +1443,66 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			CoreDNSAutoscaler:         m("gcr.io/google_containers/cluster-proportional-autoscaler:1.3.0"),
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 		},
+		"v1.15.5-rancher1-1": {
```
Comment (alena1108) [inline rke/k8s_rke_system_images.go]: we need a comment in what version of rancher it got introduced (2.2.9)

**Comment by alena1108** [comment]
Having the base copy commit here would help with review

File: rke/k8s_rke_system_images.go [hunk 2]
```diff
@@ -1063,6 +1063,34 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			CoreDNS:                   m("coredns/coredns:1.2.6"),
 			CoreDNSAutoscaler:         m("gcr.io/google_containers/cluster-proportional-autoscaler:1.0.0"),
 		},
+		"v1.13.12-rancher1-1": {
```
Comment (prachidamle) [inline rke/k8s_rke_system_images.go]: Add the comment "Enabled in Rancher <version>"

**Comment by alena1108** [comment]
LGTM

**Pull request merged**